### PR TITLE
SA key usage in examples. Helper functions in SDK

### PR DESCRIPTION
**Pull request by peter-volkov**

### CODE REVIEW

File: yandexcloud/_helpers.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+from yandex.cloud.vpc.v1.network_service_pb2 import ListNetworksRequest
+from yandex.cloud.vpc.v1.network_service_pb2_grpc import NetworkServiceStub
+from yandex.cloud.vpc.v1.subnet_service_pb2 import ListSubnetsRequest
+from yandex.cloud.vpc.v1.subnet_service_pb2_grpc import SubnetServiceStub
+
+
+class Helpers(object):
+    def __init__(self, sdk):
+        self.sdk = sdk
+        self.client = sdk.client
```
Comment (MAnyKey) [inline yandexcloud/_helpers.py]: Isn't it redundant? Can we use just SDK here?

File: yandexcloud/_helpers.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        """
+        Get ID of the first network in folder
+
+        :param folder_id: ID of the folder
+        :type operation: str
+        :return ID of the network
+        :rtype str
+        """
+        networks = self.client(NetworkServiceStub).List(ListNetworksRequest(folder_id=folder_id)).networks
+        networks = [n for n in networks if n.folder_id == folder_id]
```
Comment (MAnyKey) [inline yandexcloud/_helpers.py]: We don't need that check anymore. It is safe to remove it.

File: yandexcloud/_operation_waiter.py [hunk 3]
```diff
@@ -1,14 +1,73 @@
...
+            print_to_stream.flush()
+        time.sleep(1)
+    if print_to_stream:
+        print_to_stream.write('done')
+    return waiter.operation
+
+
+def get_operation_result(sdk, operation, response_type=None, meta_type=None, timeout=None, logger=None):
+    if not logger:
+        logger = logging
```
Comment (MAnyKey) [inline yandexcloud/_operation_waiter.py]: I'd rather use null logger that doesn't log anything.

File: yandexcloud/_sdk.py [hunk 4]
```diff
@@ -22,8 +25,14 @@ def client(self, stub_ctor, interceptor=None):
...
 
-    def waiter(self, operation_id):
-        return _operation_waiter.operation_waiter(self, operation_id, timeout=None)
+    def waiter(self, operation_id, timeout=None):
+        return _operation_waiter.operation_waiter(self, operation_id, timeout)
+
+    def wait_for_operation(self, operation_id, timeout=None, print_to_stream=None):
+        return _operation_waiter.wait_for_operation(self, operation_id, timeout, print_to_stream)
+
+    def get_operation_result(self, operation, response_type=None, meta_type=None, timeout=None, logger=None):
```
Comment (MAnyKey) [inline yandexcloud/_sdk.py]: It's not very obvious that this function will wait. Should we rename it somehow?

File: yandexcloud/_helpers.py (hunk 1 above)
Comment (peter-volkov) [reply yandexcloud/_helpers.py]: Done

File: yandexcloud/_operation_waiter.py (hunk 3 above)
Comment (peter-volkov) [reply yandexcloud/_operation_waiter.py]: Done

File: yandexcloud/_sdk.py (hunk 4 above)
Comment (peter-volkov) [reply yandexcloud/_sdk.py]: Done

File: yandexcloud/_helpers.py (hunk 2 above)
Comment (peter-volkov) [reply yandexcloud/_helpers.py]: Done

**Pull request merged**